File: peek_plugin_chat/_private/tuples/SendChatMsgActionTuple.py
from peek_plugin_chat._private.PluginNames import chatTuplePrefix
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC


@addTupleType
class SendChatMsgActionTuple(TupleActionABC):
    __tupleType__ = chatTuplePrefix + "SendChatMsgActionTuple"

    toUserId = TupleField()
    fromUserId = TupleField()
    message = TupleField()
    priority = TupleField()

File: peek_plugin_chat/plugin-module/_private/tuples/SendChatMsgActionTuple.ts
import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {chatTuplePrefix} from "../PluginNames";

@addTupleType
export class SendChatMsgActionTuple extends TupleActionABC {
    static readonly tupleName = chatTuplePrefix + "SendChatMsgActionTuple";

    stringIntId: number;

    constructor() {
        super(SendChatMsgActionTuple.tupleName)
    }
}